### PR TITLE
Connect CDAtool Cat3 export with Cypress

### PR DESCRIPTION
**Pull request by dczulada**

### CODE REVIEW

File: exporter/doc/population.go [hunk 1]
```diff
@@ -24,15 +24,15 @@ func (p Population) FindStratification(ID string) *Stratification {
...
 
 type SupplementalData struct {
-	Sex       map[string]int
-	Ethnicity map[string]int
-	Race      map[string]int
-	Payer     map[string]int
+	Sex       map[string]int 	`json:"SEX,omitempty"`
+	Ethnicity map[string]int    `json:"ETHNICITY,omitempty"`
+	Race      map[string]int    `json:"RACE,omitempty"`
+	Payer     map[string]int    `json:"PAYER,omitempty"`
```
Comment (ljades) [inline exporter/doc/population.go]: Is there a reason why the formatting for these fields (all caps) is different from most of the other json fields?

File: exporter/exporter.go [hunk 2]
```diff
@@ -276,40 +276,210 @@ func GenerateCat1(patient []byte, measures []byte, valueSets []byte, startDate i
...
 	}
 
-	data, err := AssetDir("templates/cat3/" + version)
-	if err != nil {
-		fmt.Println(err)
-	}
+	//data, err := AssetDir("templates/cat3/" + version)
+	//if err != nil {
+	//	fmt.Println(err)
+	//}
```
Comment (ljades) [inline exporter/exporter.go]: For all of these commented sections, would it make more sense to simply delete them or are we not done with that dummy code yet?

File: exporter/doc/population.go (hunk 1 above)
Comment (okeefm) [reply exporter/doc/population.go]: For whatever reason, Health Data Standards expects these population codes to be in all-caps, which I suspect is why we're doing this here.

Comment (okeefm) [reply exporter/doc/population.go]: For whatever reason, Health Data Standards expects these population codes to be in all-caps, which I suspect is why we're doing this here.

**Pull request merged**